library/src/android.rs

```diff
@@ -5,7 +5,7 @@ use std::path::{Path, PathBuf};
 
 // <https://stackoverflow.com/questions/67087597/is-it-possible-to-use-rusts-log-info-for-tests>
 #[cfg(test)]
-use std::println as debug; // Workaround to use println! for logs.
+use std::println as shorebird_debug; // Workaround to use println! for logs.
```

Comment:
We probably should have just made our own macros at this point.  Which sometimes call println and sometimes call debug!.

Reply:
Nevermind, i see we do.  I was just confused as to why we have these same std::println imports in multiple places, couldn't our macros just do that?

Reply:
Great idea! Done.